### PR TITLE
DML Implementation

### PR DESCRIPTION
**Pull request by NeelayS**
Added DML
Renamed original to vanilla
And a couple of minor corrections

### CODE REVIEW

File: KD_Lib/Bert2Lstm/bert2lstm.py [hunk 1]
```diff
@@ -41,7 +42,7 @@ def __init__(
         train_df,
         val_df,
         optimizer_teacher=None,
-        loss_fn="MSE",
+        loss_fn=nn.MSELoss(),
```
Comment (NeelayS) [inline KD_Lib/Bert2Lstm/bert2lstm.py:45]: Just a reminder that we switched from a string loss to torch module

**Comment by NeelayS** [comment]
Done

**Comment by NeelayS** [comment]
Done

**Comment by Het-Shah** [comment]
I hope you are doing basic benchmarking to test if all the algorithms are training properly. I am not testing the code.

**Comment by NeelayS** [comment]
Yes, I'm trying them out on MNIST for 2-3 epochs on MNIST to check that they're working

**Comment by Het-Shah** [comment]
Cool, let me know what the results are.

**Comment by NeelayS** [comment]
Should we maintain a markdown file in the repo to log results obtained when adding new implementations ?

**Comment by Het-Shah** [comment]
Yup that is the plan to add benchmarking.

**Comment by Het-Shah** [comment]
Yea seems good to me.

**Comment by Het-Shah** [comment]
Merging.

**Pull request merged**